src/main/java/org/prebid/server/handler/CurrencyRatesHandler.java

```diff
@@ -0,0 +1,48 @@
+package org.prebid.server.handler;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Handler;
+import io.vertx.core.json.Json;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.RoutingContext;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.prebid.server.currency.CurrencyConversionService;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+/**
+ * Handles HTTP request for currency latest update lastUpdate.
```

Comment:
Please rephrase this sentence as follows:
"Handles HTTP request for latest currency rates update information"